presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/CanonicalizeExpressionRewriter.java

```diff
@@ -25,8 +25,12 @@
 import com.facebook.presto.sql.tree.QualifiedName;
 import com.google.common.collect.ImmutableList;
 
+import java.util.List;
+
 public class CanonicalizeExpressionRewriter
 {
+    private static final List<QualifiedName> rewriteFns = ImmutableList.of(QualifiedName.of("map_concat"), QualifiedName.of("array_concat"));
```

Comment:
Don't use shortform for variable names. May be use `rewriteSupportedFunctions` or `rewriteEnabledFunctions` ?
Checkout formatter section in
https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines

Reply:
made changes